index.js

```diff
@@ -222,7 +233,9 @@ module.exports = (function () {
                 return retry_loop();
               }
 
-              save_response_in_cache(options.method, options.url, options.auth.bearer, res, body);
+              if (res.statusCode >= 200 && res.statusCode < 300){
+                save_response_in_cache(options.method, options.url, options.auth.bearer, res, body);
```

Comment:
Hmm could make that configurable in the future? I can't think of a use case for us wanting to cache 404s